katsdpdata/prod_handler.py

```diff
@@ -22,8 +24,8 @@
 logger = logging.getLogger(__name__)
 
 CAPTURE_BLOCK_REGEX = "^[0-9]{10}$"
-CAPTURE_STREAM_L0_REGEX = "^[0-9]{10}[-_].*l0$"
-CAPTURE_STREAM_L1_REGEX = "^[0-9]{10}[-_].*l1-flags$"
+CAPTURE_STREAM_L0_REGEX = "^[0-9]{10}[-_].*l0*$"
```

Comment:
Why did you add this * here?

Reply:
I added the symbol * because incomplete rdb files for continuum products were not considered pruned products.